### PR TITLE
Fill in recent 'since' annotations

### PR DESCRIPTION
**Pull request by NotMyFault**
```
Analyzing core/src/main/java/jenkins/util/SetContextClassLoader.java:51
        first sha: f77a28b37adcacdce40ffecc0544df104d708d55
        first tag was jenkins-2.362
        Updating file in place
Analyzing core/src/main/java/jenkins/util/SetContextClassLoader.java:62
        first sha: f77a28b37adcacdce40ffecc0544df104d708d55
        first tag was jenkins-2.362
        Updating file in place
Analyzing core/src/main/java/jenkins/util/SetContextClassLoader.java:73
        first sha: f77a28b37adcacdce40ffecc0544df104d708d55
        first tag was jenkins-2.362
        Updating file in place
Analyzing core/src/main/java/jenkins/util/SetContextClassLoader.java:84
        first sha: f77a28b37adcacdce40ffecc0544df104d708d55
        first tag was jenkins-2.362
        Updating file in place
Analyzing war/src/main/java/executable/Main.java:81
        first sha: ed0ce3877c1d1ee5d078f48a87dbe958514a48ff
        first tag was jenkins-2.358
        Updating file in place
Analyzing war/src/main/java/executable/Main.java:92
        first sha: ed0ce3877c1d1ee5d078f48a87dbe958514a48ff
        first tag was jenkins-2.358
        Updating file in place

List of commits introducing new API and the first release they went in:
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.358
  - https://github.com/jenkinsci/jenkins/commit/ed0ce3877c1d1ee5d078f48a87dbe958514a48ff
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.362
  - https://github.com/jenkinsci/jenkins/commit/f77a28b37adcacdce40ffecc0544df104d708d55
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7059"></a>

### CODE REVIEW

**Review by basil — APPROVED** [review]

File: war/src/main/java/executable/Main.java
Comment (daniel-beck) [inline war/src/main/java/executable/Main.java:81]:
```suggestion
     * @since 2.66
```

https://github.com/jenkinsci/extras-executable-war/commit/7afa0adbd7a6b0f1391e7ebd3213e4a79ae45e3f / https://github.com/jenkinsci/jenkins/commit/cb3f713122030b0ec5097b286d4e9e566a34a90a

File: war/src/main/java/executable/Main.java
Comment (daniel-beck) [inline war/src/main/java/executable/Main.java:92]:
```suggestion
     * @since 2.66
```

https://github.com/jenkinsci/extras-executable-war/commit/7afa0adbd7a6b0f1391e7ebd3213e4a79ae45e3f / https://github.com/jenkinsci/jenkins/commit/cb3f713122030b0ec5097b286d4e9e566a34a90a

**Review by daniel-beck — CHANGES_REQUESTED** [review]

**Comment by NotMyFault** [comment]
How cruel

**Review by daniel-beck — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
